rules/opentitan_test.bzl

```diff
@@ -151,7 +151,7 @@ def cw310_params(
             "console",
             "--exit-success={exit_success}",
             "--exit-failure={exit_failure}",
-            "--timeout=3600s",
+            "--timeout=10s",
```

Comment:
Was this supposed to go into the commit? Seems like the kind of change you'd do in your local repo, for convenience.

Reply:
I think this is something we want, I don't think we have any fpga tests that come close to the 1hr mark.

Reply:
This was set to be 3600s to be compatible with the maximum test timeout associated with bazel timeouts.

It may be better to just leave this unset and leave bazel to kill tests when they reach their timeouts.

Reply:
I agree, done.